scrapli_community/huawei/smartax/huawei_smartax.py

```diff
@@ -60,12 +71,14 @@
         "async_on_open": default_async_on_open,
         "sync_on_close": default_sync_on_close,
         "async_on_close": default_async_on_close,
-        "failed_when_contains": ["Error:"],
-        "textfsm_platform": "huawei_vrp",
+        "failed_when_contains": [
+            "% Ambiguous command",
+            "% Incomplete command",
+            "% Invalid input detected",
+            "% Unknown command",
+            "Error:",
+        ],
+        "textfsm_platform": "huawei_smartax",
         "genie_platform": "",
-        # Force the screen to be 256 characters wide.
-        # Might get overwritten by global Scrapli transport options.
-        # See issue #18 for more details.
-        "transport_options": {"ptyprocess": {"cols": 256}},
```

Comment:
was this removed on purpose? it probably cant hurt and I recall it being "needed" for at least some boxes so we dont wanna brick that functionality ideally.

Reply:
ah. I see your comment now. are you suuuuuure? I am pretty sure we can find an issue from before (maybe in scrapli core?) on this. did you test w/ ssh2 only or also system? this would only be relevant for system things.

Reply:
I removed it because i saw that the issue #18 is related to Huawei VRP. These SmartAX devices use other OS. I didn't test enough with system. Will do more test and comment back later.

Reply:
Changes looks good to me, thanks for patching @codingnetworksb !